src/components/Article.jsx

```diff
@@ -1 +1,34 @@
-// export const Article = () => ();
+import React from 'react';
+
+export const Article = ({ title = '', date = '', text = '' }) => {
+  const dateISO = date.toISOString().slice(0, 10);
+  const dateLocaleString = date.toLocaleDateString('en-US', {
+    year: 'numeric',
+    month: 'long',
+    day: 'numeric',
+  });
+
+  return (
+    <div className="section">
+      <div className="container">
+        <h1 className="title is-2">React Article</h1>
```

Comment:
this part shouldn't be here, check where the comment starts in App.js
In general components should be as self-contained and small as possible